pkg/remotemount/remotemount.go

```diff
@@ -0,0 +1,82 @@
+package remotemount
```

Comment:
We should mount the 9p mounts ASAP, before establishing the SSH connection.

Can we just dynamically generate `/etc/fstab` entries in https://github.com/lima-vm/lima/tree/master/pkg/cidata/cidata.TEMPLATE.d/boot ?
Run `mount -o remount` if the filesystem is already mounted.

Reply:
We are generating the /etc/fstab, this is just a fallback for existing virtual machines being edited.